src/view/components/customerIdWrapper.jsx

```diff
@@ -0,0 +1,186 @@
+import "regenerator-runtime"; // needed for some of react-spectrum
```

Comment:
In hindsight, I guess I could've left this in setCustomerIds.jsx. This is a result of my trying to figure out how to implement the name space API call.

Reply:
I mean customerIdWrapper.jsx .